README.md

```diff
@@ -1 +1,2 @@
-# code-review-project
\ No newline at end of file
+# tributepage to John Maccarthur. 
+##### in freecodecamp's project
```

Comment:
Make your README.md file more elaborate so as to give a good description of your project. Think of add installation processes even if it means a simple way of cloning and running the repository in one's local machine.